decompressed/gui_file/www/lang/it-it/webui-core.po

```diff
@@ -2034,7 +2034,7 @@ msgid "Setup Wizard"
 msgstr "Configurazione guidata"
 
 msgid "Short Guard Interval"
-msgstr "Short Guard Interval"
+msgstr "Intervallo controllo breve"
```

Comment:
non ha alcun senso tradutto in italiano, cosi come altri termini tecnici che son qui son semplicemente da lasciare in inglese

Reply:
@FrancYescO 

Non essendo un acronimo o qualcosa di specifico tecnico ma è la definizione di un parametro potrebbe essere tradotto.

Se vuoi lasciarlo così in inglese per me va bene.

Reply:
Non sono d'accordo sulle modifiche che hai fatto.
Tu mi avevi indicato solo una stringa.
Se mi avessi indicato anche le altre ti avrei detto che non ero d'accordo.

Capisco voler mantenere i termini tecnici , se ci sono, in inglese (ma non sempre per me è necessario...),

Per favore non sono d'accordo che questo sia un termine "tecnico" 

**_"Force radio down"_** 

è un **termine assolutamente descrittivo  non tecnico** o che altro. Per me la traduzione 

**_"Forza radio OFF"_** 

era assolutamente corretta e consona e meglio della stringa "Force radio down" che in un'interfaccia in italiano non ha nessun senso.

Reply:
Come già spiegato non c'è nulla di descrittivo, è una funzione, nemmeno tanto documentata (e come questa ce ne sono tante altre tradotte dove non ce ne era necessitá di traduzione italiana, ne ho risostituite solo alcune che mi son capitate sotto occhio)